### ISSUE TITLE
What is the purpose of the input text box on the main page of the replay interface?

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#406]
_I_ know this but it is not immediately apparent in the interface of how the users should use this text box. Entering a value makes it more clear but it ought to be self-evident in the main interface.

**Comment by machawk1** [comment]
<img width="1069" alt="screen shot 2018-06-29 at 5 44 38 pm" src="https://user-images.githubusercontent.com/2514780/42116228-5beb2316-7bc4-11e8-97ad-1e800984b1cf.png">

**Comment by ibnesayeed** [comment]
The placeholder text `http://` can be changed to something like `Search a URL in the archive`.

**Comment by machawk1** [comment]
@ibnesayeed That is what I was thinking as well. I recall you recently having a similar conversation with @shawnmjones on this placeholder text. Would having a submission button help with the usability/accessibility/clarity of the text box's function in addition to modifying the placeholder text?

**Comment by ibnesayeed** [comment]
Having a call for action button is not a bad idea. In that case the placeholder text can be a sample URL (e.g., `http://example.com/`), but not just `http://` and the text describing the purpose of the form (e.g., `Search a URL in the archive`) can go to the button.

**Comment by machawk1** [comment]
I believe this suggestion is more accessible, as it is providing an example of what should be entered as the input instead of instruction as the placeholder.

Based on the above I am proposing:
* Adding a label (again, for accessibility)
* Replacing the "http://" with "http://example.com"
* Adding a submit button (potentially with the value text provided)

Do we anticipate any other UI elements on this page beyond those associated with this text box?

The bottom portion of the page is admittedly a mess and won't scale, but is useful for users trying out the app. Adjusting the availability (or lack thereof) of the admin interface from this page is addressed in another ticket, so some refactoring is in order there, too.

**Comment by ibnesayeed** [comment]
If the text field is semantically named and the button text is descriptive enough then an additional label will be a noise without mush accessibility benefits.

**Comment by machawk1** [comment]
@ibnesayeed Are you proposing we _not_ have a label for the text input? Is using the name attribute for the text input sufficient for it be considered accessible a la Section 508?

**Comment by ibnesayeed** [comment]
I was suggesting we not have a visible label text, but we can always include attributes like `title` and `aria-label` for better accessibility on non-visual media.
